dip.schema.json

```diff
@@ -123,6 +123,9 @@
                 "full",
                 "delta"
               ]
+            },
+            "useTimestampForQuery": {
```

Comment:
It's weird to have a boolean that defaults to true when not given. Can this be something like disableTimestamp, ignoreQueryTimestamp, etc.?

Reply:
Good point. I've updated the field to ignoreQueryTimestamp.